FAQs + installation french translation (#189)
BeamMP/Docs: docs/fr/game/getting-started.md

# Pour commencer

## **1. Compatibilitée**

BeamMP est entièrement compatible avec Windows et Linux, la compatibilité avec MacOS est en cours de développement. 
Cependant, puisque Linux et MacOS sont des systèmes d'opération secondaire plusieurs problèmes (bug) sont à prévoir.

!!!warning

>     BeamMP ne fonctionnera pas avec une version piratée ou dépassée de BeamNG.DRIVE. 

Reviewer: ```suggestion
    BeamMP ne fonctionnera pas avec une version piratée ou dépassée de BeamNG.drive.
```

6. Une fois que BeamNG est lancé, cliquer sur l'onglet `Repository` (Dépot en français) dans le menu principal et assurez-vous que `multiplayerbeammp` **est le seul** mod activé.
7. Retournez au menu principal and cliquez le bouton 'Multiplayer' pour lancer le mode multijoueur.
8. On va vous demander si vous voulez vous connecter ou si vous voulez jouer en teant qu'invité (guest) (Certains serveurs bloque les joueur invités (guest)). Vous pouvez créer un compte sur notre [forum](https://forum.beammp.com) et vous connecter à BeamMP avec les mêmes informations par la suite.
9. Choisissez n'importe quel serveur de votre choix et appuyer sur `Connect`. Amusez-vous!

!!!note

      Si vous vous connectez à un serveur avec plusieurs joueurs et/ou véhicules, les temps de chargement peuvent s'allonger.

### **2b. Installation sur Linux**

Vous devrez construire le lanceur vous même pour le moment.
Pour ce faire, vous avez besoin d'un minimum de connaissences de base pour construire (build) une application.

> Assurez-vous d'avoir [`vcpkg` installed](https://learn.microsoft.com/en-us/vcpkg/get_started/get-started?pivots=shell-bash#1---set-up-vcpkg) d'installé, ainse que des outil de dévellopement de base, souvent trouvé dans des packets (package) tel que (par exemple):

Reviewer: ```suggestion
Assurez-vous d'avoir [`vcpkg`](https://learn.microsoft.com/en-us/vcpkg/get_started/get-started?pivots=shell-bash#1---set-up-vcpkg) d'installé, ainse que des outil de dévelopement de base, souvent trouvé dans des packets (package) tel que (par exemple):
```

- Debian: `sudo apt install build-essential`
- Fedora: `sudo dnf install @development-tools`
- Arch: `sudo pacman -S base-devel`
- openSUSE: `zypper in -t pattern devel-basis`

> - SteamOS (Arch): `sudo pacman -S base-devel linux-api-headers glibc libconfig` (You also need to do `sudo steamos-readonly disable` but make sure to enable it again after installing the packages)

Reviewer: ```suggestion
- SteamOS (Arch): `sudo pacman -S base-devel linux-api-headers glibc libconfig` (Vous devez également faire `sudo steamos-readonly disable` mais assurez-vous de l'activer à nouveau après l'installation des paquets.)
```

Clônez le répertoire BeamMP-Launcher sur votre système en utilisant `git`, exemple:
`git clone https://github.com/BeamMP/BeamMP-Launcher.git`

> [Additional information about cloning a GitHub Repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)

Reviewer: ```suggestion
[Informations complémentaires sur le clonage d'une réplique GitHub](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
```

- Notez l'emplacement du userfolder de Proton-BeamNG.drive  (il est généralement trouvé dans `~/.local/share/Steam/steamapps/compatdata/284160/pfx/drive_c/users/steamuser/AppData/Local/BeamNG.drive`)
- Créez un symlink entre les deux userfolders `ln -s ~/.local/share/Steam/steamapps/compatdata/284160/pfx/drive_c/users/steamuser/AppData/Local/BeamNG.drive ~/.local/share`

Avec le symlink configuré antre le lanceur compilé et les userfolders, vous pouvez lancer le jeux avec Proton via Steam tout en executant automatiquement le lanceur avec les remplacement suivents pour les option de lancement du jeu vanilla (version sans mod), retrouvés dans la fenêtre des propriétées du jeu dans son entrée sur Steam.

- `~/BeamMP/BeamMP-Launcher --no-launch & %command% ; killall BeamMP-Launcher`

Notez que ceci assume que vous avez mis, les binairies du lanceur que vous avez compilé plus tôt, dans `/home/user/BeamMP/`, donc changez l'emplacement pour qu'il corresponde à où vous avez mis les binairies terminées et vous allez avoir besoin de recompiler le lanceur avec la bonne branche git à chaque fois qu'une nouvelle mise à jour sera disponible.

> !!! tip "Ajoutez une police d'écriture  pour les emojis pour avoir les emojis en jeu"

Reviewer: ```suggestion
!!! tip "Ajoutez une police d'écriture pour les emojis pour avoir les emojis en jeu"
```
